src/ray/stats/stats_test.cc

```diff
@@ -0,0 +1,34 @@
+#include "gmock/gmock.h"
```

Comment:
This test is never being built or run in our CI, right?

Reply:
yeah. I'll refine it soon. either remove it or add some assert and enable it in CI.